deps/llvm.mk

```diff
@@ -175,6 +175,19 @@ LLVM_CPPFLAGS += -flto
 LLVM_LDFLAGS += -flto
 endif # LLVM_LTO
 
+ifeq ($(BUILD_CUSTOM_LIBCXX),1)
+LLVM_LDFLAGS += -Wl,-R$(build_libdir)
+LLVM_CXXFLAGS += -I$(build_includedir)
+LLVM_CPPFLAGS += -I$(build_includedir)
+LLVM_CFLAGS += -I$(build_includedir)
```

Comment:
Confusingly, `CPP` stands for `C PreProcessor`, not `C Plus Plus`.  `CXX` on the other hand, is `C++`, but with the `+` symbols turned on their side.

What on God's Green Earth were the programmers thinking.